src/content/plugins/context-replacement-plugin.md

```diff
@@ -31,10 +31,17 @@ If the resource (directory) matches `resourceRegExp`, the plugin replaces the de
 Here's a small example to restrict module usage:
 
 ```javascript
+// moment.js
 new webpack.ContextReplacementPlugin(
   /moment[/\\]locale$/,
   /de|fr|hu/
 );
+
+// date-fns
+new ContextReplacementPlugin(
+  /date\-fns[\/\\]/, 
+  /[/\\](de|fr|hu)[/\\]/
+  );
```

Comment:
Please fix indentation.

Reply:
I believe it's now correct

Reply:
I haven't seen you reply but now I do believe it's correct. Do you have any other things you'd like to point out?